eng/pipelines/scripts/mgmt_sdk_compiler.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+echo "CWD : "
+pwd
+#TODO:
+#for some reason the workingdirectory dos not seem to work...
+#fix the following cd cmd once we figure out how to get it to work
+#change to the root of the sources repo
+cd ../../..
+
+for i in `ls -d */*/v20* | grep -v "node_modules/*/*"`; 
+do 
+  echo "building folder $i"
+  cd $i; 
+  mvn clean compile -Dgpg.skip; 
```

Comment:
If you are going to do this in a script I highly recommend you pass through all the options to this mvn call to allow it to call other targets like and other options as needed.

Reply:
all thes pipeline is supposedto do is to do a compile on all the sdks.. i have moved all the appropriate options over to the script